blocks/init/src/Blocks/components/icon/icon-style.scss

```diff
@@ -0,0 +1,45 @@
+$icon: (
+	colors: (
+		primary: (
+			normal: (
+				color: global-settings(colors, primary),
+			),
+			hover: (
```

Comment:
Do all icons need to have a hover effect by default?

They're all set to `currentColor` for fills/strokes, so for most cases regular color option is more than enough.
For cases when hover style change is needed the icon's color can be easily overridden.

Reply:
Makes sense..no need for all color variations. I changed it by leaving only primary and black color.

Reply:
Sorry, looks like there was a misunderstanding. The multiple color variants were fine (even allowing all manifest colors would be fine).

I was asking about the need for a (default) hover state for the icon. In most cases the icon will be either a decoration by itself or a part of something interactive (like a button).

If the icon is a part of something else (e.g. a Button), because the SVGs for all the icons have `currentColor` as their fill/stroke colors, they would pick up the surrounding text color (can also be set as `color: inherit` in CSS).
By inheriting the text color, you don't need to worry about changing the hover/focus/disabled/etc. color because that would be handled by the parent component.

In cases where the icon should be decorative or by itself (e.g. a Card or List), a color option is useful, but in those cases the hover color probably wouldn't be needed.

Reply:
Good point. This was taken from my implementation because I had a case where only an icon was used so I needed a hover. I'll have to change this implementation, as the hover looks odd when used in a compound setting (as a part of another block). 👍🏼

Reply:
Ok, would it be fine with you that I remove the color options from the component and just set `color: inherit` inside the css?

Reply:
Because I think the component will mostly be used as part of a parent component.

Reply:
Good idea!